packages/dara-components/dara/components/graphs/graph_layout.py

```diff
@@ -57,6 +57,78 @@ class GraphLayout(BaseModel, abc.ABC):
     node_font_size: Optional[int] = None
 
 
+class TiersConfg(BaseModel):
+    """
+    TiersConfg provides a way of defining tiers for a graph layout.
+
+    :param group: Path within node to group property which defines the tier it belong to
+    :param order_nodes_by: A path to a node property which contains a number defining the order of nodes within a tier
+    :param rank: A list of grouo values defining the order they should appear in
```

Comment:
Might be worth adding a very short example in the docstring so users can understand what these mean at a glance, I know you have a long one in TieringLayout but users likely won't see the docstring in their editor as this is just a base class. TiersConfig is a class they will interact with so could be useful to add